docs/source/techniques/ReflectometryILL.rst

```diff
@@ -0,0 +1,519 @@
+
+*ForegroundHalfWidth* is a list of one or two values. If a single value is given, then this number of pixels on both sides of the centre pixel are included in the foreground. For example, ``ForegroundHalfWidth=[3]`` means three pixel on both sides are included, making the foreground seven pixels wide in total.
+``ForegroundHalfWidth=[0]`` means that only the centre pixel is included. When two values are given, then the foreground is asymmetric around the centre. For instance, ``ForegroundHalfWidth[2,5]`` indicates that two pixel at lower :math:`\theta` and five pixels at higher :math:`\theta` are included in the foreground.
+
+*LowAngleBkgWidth* and *HighAngleBkgWidth* define the number of the background fitting pixels at low and high :math:`\theta`. Either one or both widths can be defined. The distance between the background pixels and the foreground can in turn be given by *LowAngleBkgOffset* and *HighAngleBkgOffset*.
+
+
+Gravity correction
+------------------
+
+Gravity correction is relevant for data reduction at FIGARO. Its execution is steered by a switch: `CorrectGravity`. Gravity correction approach follows the algorithm described
+in Ref. [#Gutfreund]_, where both the wavelength and reflection angles are corrected for the neutron drop due to gravity. This approach assumes no significant change to the neutron speed.
+
+In practice, this correction had to be split in two steps. Both the new wavelength axis and the corrected reflected angle distribution are calculated during the preprocessing, both
+for direct and reflected beams. The corrected wavelength axis is applied immediately in the preprocess algorithm, while the reflected angle correction is applied only to the reflected beam
+workspace at a later stage, in the :ref:`ReflectometryILLConvertToQ <algm-ReflectometryILLConvertToQ>` algorithm, immediately before the final conversion to the momentum exchange. This approach
+was dictated by the Mantid architecture but is mathematically equivalent and does not impact the quality of the correction.
+
+Summing foreground
+==================
+
+The default summation type is incoherent (sum along constant :math:`\lambda`), where the reflectivity curve is calculated by dividing the summed foreground of the reflected beam by the summed foreground of the direct beam. For coherent summing, first the reflected beam data is divided by the direct beam data in 2D, then the ratio is summed along the lines of the constant :math:`Q_{z}`.
+
+Polarisation correction
+=======================
+
+This algorithm is an ILL specific wrapper around :ref:`PolarizationEffiencyCor <algm-PolarizationEfficiencyCor>`. It is typically run in reflectometry reduction workflow, namely the workflow in the :ref:`ReflectometryILLAutoProcess <algm-ReflectometryILLAutoProcess>` algorithm, between :ref:`ReflectometryILLSumForeground <algm-ReflectometryILLSumForeground>` and :ref:`ReflectometryILLConvertToQ <algm-ReflectometryILLConvertToQ>`.
```

Comment:
What is "this algorithm" here? Is this ReflectometryILLPolarizationCor? Perhaps worth making this clear